src/tokenizer.ts

```diff
@@ -425,12 +429,20 @@ export class Tokenizer {
     }
   }
 
-  enqueueLookahead(tok: Token, pos: number) {
+  getLocation() {
```

Comment:
```suggestion
  getLocation(): Position {
```

Useful if you decide to rename or add a new member.